### ISSUE TITLE
Strong type issue flagged with empty defaultValue string

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#1100]
**Description of the issue**

When an empty string is used for the default value of a parameter strong type fails.

While this is true, an empty string is often used as a placeholder when creating optional parameters.

**Expected behaviour**

Don't flag a strong type issue, when the parameter `defaultValue` is configured as an empty string.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.10.0-B2111058**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.2.0
PSEdition                      Core
GitCommitId                    7.2.0
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
